services/cd-service/Makefile

```diff
@@ -95,7 +98,9 @@ release: docker
 
 all: test docker
 
-publish:
-	echo "nothing to do"
+login:
```

Comment:
This should probably be marked `.PHONY`?

Reply:
is publish is never created as a file, would phony matter?